docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
+---
+
+The `fx` extension executes an Edge Worker.
+
+## Syntax
+
+Executes parameterless function:
+
+    <OBJECT> fx(<STRING> name)
+
+Executes function with parameters set via JSON or object:
```

Comment:
I recommend putting each of these different functions in a different section under syntax with an h3 heading. This will make it easier for readers to scan.

Also, consider putting the code strings in code blocks. That will make it easier to copy/paste. I also like to have examples with actual code besides the syntax strings. Something like:

Here is the syntax, use it for blah blah blah:
`code string here`

An example that does this thing with a function named blah and parameters blah blah would look like:
`fake example here`

Reply:
Changed to code block